api/v1beta1/machinedeployment_types.go

```diff
@@ -211,6 +216,31 @@ type MachineRollingUpdateDeployment struct {
 
 // ANCHOR_END: MachineRollingUpdateDeployment
 
+// ANCHOR: RemediationStrategy
+
+// RemediationStrategy allows to define how the MachineSet can control scaling operations.
+type RemediationStrategy struct {
+	// MaxInFlight determines how many in flight remediations the MachineSet should issue at
+	// any given point in time.
+	//
+	// This field gives control over how many Machines should be remediated (deleted),
+	// at any given point in time during the MachineSet's reconciliation.
+	//
+	// Remediation only happens on the MachineSet with the current revision, during rollout
+	// operations, older MachineSets aren't allowed to remediate while a rollout is in progress.
+	// Unhealthy machines are always prioritized during scale down operations over healthy ones.
+	//
+	// MaxInFlight can be set to a fixed number or a percentage.
+	// Example: when this is set to 20%, the MachineSet controller deletes at most 20% of
+	// the desired replicas.
+	//
+	// If not set, remedation has an unbounded limit.
```

Comment:
```suggestion
	// If not set, remediation has an unbounded limit.
```
nit: technically not unbounded :) I guess max is the number of existing machines (no need to udpate this though)

Reply:
reworded a bit